Source/Plugins/ImportExport/ImportExportFilters/NodesTrianglesToVtk.cpp

```diff
@@ -211,8 +204,7 @@ void NodesTrianglesToVtk::execute()
   {
 
     QString ss = QObject::tr(": Error opening Triangles file '%1'").arg(m_TrianglesFile);
-    setErrorCondition(-1);
-    notifyErrorMessage(getHumanLabel(), ss, -666);
+    setErrorCondition(-667, ss);
```

Comment:
This error code is not consistent with either the error code set or what the previous message emitted

Reply:
See comment above.

Reply:
I wrote the original code and reviewed the current state of the code. I am ok with changing the -1 to -666